build/Microsoft.DotNet.Cli.BundledSdks.proj

```diff
@@ -0,0 +1,46 @@
+<Project ToolsVersion="15.0" DefaultTargets="CopySdkToOutput">
+  <!-- workaround for https://github.com/Microsoft/msbuild/issues/885 -->
+  <!-- renaming the property because the original property is a global property and therefore
+       cannot be redefined at runtime. -->
+  <PropertyGroup>
+    <CLIBuildDllPath>$([MSBuild]::Unescape($(CLIBuildDll)))</CLIBuildDllPath>
+  </PropertyGroup>
+
+  <UsingTask TaskName="DotNetRestore" AssemblyFile="$(CLIBuildDllPath)" />
+
+  <Target Name="CopySdkToOutput"
+          DependsOnTargets="PrepareBundledSdksProps;
+                            EnsureSdkRestored;
+                            GetSdkItemsToCopy"
+          Inputs="@(SdkContent)"
+          Outputs="@(SdkContent->'$(oz)/$(SdkPackageName)/$(SdkPackageVersion)/%(RecursiveDir)%(FileName)%(Extension)')">
+    <Copy SourceFiles="@(SdkContent)"
+          DestinationFiles="@(SdkContent->'$(SdkLayoutDirectory)/%(RecursiveDir)%(FileName)%(Extension)')" />
+  </Target>
+
+  <Target Name="GetSdkItemsToCopy">
```

Comment:
CLI's use of MSBuild was built up before we had proper project support and we developed some inconvenient patterns. I'm maintaining those patterns [orchestrator target like `CopySdkToOutput`] instead of proper dependency chains. When we get time to design how we want to structure our build, this should make it easier to convert everything at once instead of having yet-another-pattern that wasn't fully designed. Perhaps you will be able to help us design a restructure plan some time.